### PR TITLE
Imagic msa

### PR DESCRIPTION
**Pull request by delarosatrevin**
* Added package for IMAGIC protocols (MSA workflow)
Some bugfixes:
* Fixed bug in apply alignment using matrix (Python binding)
* Fixed bug in Spider (now applying alignment if exists)
* Back-ported (from scipion-box) some utilities to use EMAN for some image conversions.

### CODE REVIEW

File: pyworkflow/tests/em/data/test_data.py [hunk 1]
```diff
@@ -86,6 +87,15 @@ def testExistLocation(self):
...
         self.assertTrue(ih.existsLocation(newFn))
 
+    def test_convertMicrographs(self):
+        """ Convert micrograhs to different formats.
+         EMAN2 required for .img
+        """
+        micFn = self.dataset.getFile('micrographs/BPV_1386.mrc')
+        outFn = pwutils.replaceBaseExt(micFn, 'img')
+        ih = ImageHandler()
+        ih.convert(micFn, os.path.join('/home/josem', outFn))
```
Comment (josuegbl) [inline pyworkflow/tests/em/data/test_data.py]: directory /home/josem/ not exists always. The path to do the convert must be more general

File: pyworkflow/tests/em/data/test_data.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ def testMd(self):
         self.assertEqual(xList,x)
         self.assertEqual(yList,y)
```
Comment (josuegbl) [inline pyworkflow/tests/em/data/test_data.py]: thre are any test for .dm4 format? It seems the test not view this condition.

**Comment by josuegbl** [comment]
Review the comments in the code.

**Comment by delarosatrevin** [comment]
@josuegbl , check again and pass the added tests.

**Comment by josuegbl** [comment]
everything ok.... imagehandler tests passed.

**Pull request merged**